packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -453,8 +453,68 @@ class _ModalBottomSheetState<T> extends State<_ModalBottomSheet<T>> {
   }
 }
 
-class _ModalBottomSheetRoute<T> extends PopupRoute<T> {
-  _ModalBottomSheetRoute({
+/// Shows a modal Material Design bottom sheet.
+///
+/// A modal bottom sheet is an alternative to a menu or a dialog and prevents
```

Comment:
It looks like there is a fair bit of duplication between this class and showModalBottomSheet
Can you use `@template` to avoid duplication?

Reply:
Hey how would I exactly use the template for these docs? Should I use one template for multiple parameters / paragraphs or one template per parameter?

Reply:
cc @chunhtai

Reply:
I will try to group them into one shareable paragraph